main.go

```diff
@@ -52,12 +70,10 @@ func _main() int {
 
 	if err != nil {
 		log.Error("initialization failure", err)
-		return 1
```

Comment:
os.Exit causes the defer log.Flush() not to run, hence the exit status and subfunction.